pkgs/applications/misc/cura/plugins.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, python3Packages }:
+
+let
+
+  self = {
+
+    octoprint = stdenv.mkDerivation rec {
+      name = "Cura-OctoPrintPlugin";
+      version = "3.5";
+
+      src = fetchFromGitHub {
+        owner = "fieldOfView";
+        repo = "Cura-OctoPrintPlugin";
+        rev = version;
```

Comment:
In the `Cura-OctoPrintPlugin` repository, `3.5` is a branch and not a tag.  That is, it changes regularly and does not point to a fixed commit.  Please just put the latest revision here.